pkg/tnf/test.go

```diff
@@ -33,6 +34,19 @@ const (
 	FAILURE
 )
 
+// TestsExtraInfo a collection of messages per test that is added to the claim file
+// use WriteTestExtraInfo for writing to it
+var TestsExtraInfo []map[string][]string = []map[string][]string{}
```

Comment:
I believe we should have an opinion about the payload, rather than `[]string`.  Instead, can we use something like this as the payload:

```
type ExtraInfo struct {
  messages string
  timestamp Time
}
```

and consequently:

```
var TestsExtraInfo []map[string]ExtraInfo = []map[string]ExtraInfo{}
```

Instead of stacking messages, call `WriteTestExtraInfo(...)` for each invocation, so each message is timestamped, similar to the information we would get from `logrus`.

Reply:
I do not see what value do u get from the timestamp. The 'problem' we see with the CNF is not related to a point in time and the messages are for general information and not logging.
There's a reason for providing a function that takes []string and not just string. I'll explain. We said that the test does not have a unique key. The ginkgo.FullSpecText can be the same for two instances of the same test. That's why we moved from a map of info to a slice of info. Ignore the current inner map. I'm referring to []map instead of map[]. Between calls to WriteExtraInfo, I do not know to which test to add the message. 2 options. One is letting the user supply a test object that he/she keeps locally through the test. Second, let the user store a slice locally and call WriteExtraInfo with the slice. I think option #2 is better for the user.